includes/embeds/class-amp-facebook-embed.php

```diff
@@ -136,7 +84,13 @@ public function sanitize_raw_embeds( Document $dom ) {
 				$scripts[] = $script;
 			}
 			foreach ( $scripts as $script ) {
-				$script->parentNode->removeChild( $script );
+				$parent_node = $script->parentNode;
+				$parent_node->removeChild( $script );
+
+				// Remove parent node if it is an empty <p> tag.
+				if ( 'p' === $parent_node->tagName && null === $parent_node->firstChild ) {
```

Comment:
The `script` tag is wrapped with a `p` tag (from `wpautop()`), so I've removed the lingering empty tag if it's there.